src/pages/Results/Results.page.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import VideoList from '../../components/VideoList';
+import useAPI from '../../hooks/useAPI';
+import useQueryParams from '../../hooks/useQueryParams';
+import { Container, Title } from './Results.styles';
+
+const ResultsPage = () => {
+  const queryParam = useQueryParams().searchedText;
+  const [videos, loading] = useAPI(queryParam);
+
+  return (
+    <Container>
+      <Title data-testid="results-message">Results for {`"${queryParam}"`}: </Title>
```

Comment:
You could reach this resultant object by role: `header` and get rid of the `data-testid` attribute